### PR TITLE
91 remove hello worlds from code

### PR DESCRIPTION
**Pull request by julianhahn**
cloeses #91

### CODE REVIEW

**Comment by julianhahn** [comment]
@anbraten created a draft!

**Comment by anbraten** [comment]
@julianhahn Code look fine to me. Have you re-build the commons with `yarn build:dep`?

**Comment by julianhahn** [comment]
> @julianhahn Code looks fine to me. Have you re-build the commons with `yarn build:dep`?

Did a yarn build. Is build:rep there included?

**Comment by anbraten** [comment]
Yeah that should be triggered by that command as well.

**Comment by julianhahn** [comment]
Here's my terminal till the freeze point. Maybe this helps?
grafik

**Comment by anbraten** [comment]
@julianhahn Probably fixed it. Please try merging master.

**Comment by julianhahn** [comment]
very nice! It's starting now.

File: packages/app/src/lib/feathers.ts [hunk 1]
```diff
@@ -17,10 +10,6 @@ type Service<T> = AdapterService<T> & ServiceAddons<T>;
 interface ServiceTypes {
   users: Service<User>;
   'nfc-readers': Service<NFCReader>;
```
Comment (anbraten) [inline packages/app/src/lib/feathers.ts:12]: Could you add the `nfc-tags` here as well.

**Review by anbraten — CHANGES_REQUESTED** [review]
LFTM

File: packages/app/src/lib/feathers.ts (hunk 1 above)
Comment (julianhahn) [reply packages/app/src/lib/feathers.ts:12]: fixed in 945a2944136251cfba9c25f265a43eaeaabfad79

**Review by anbraten — APPROVED** [review]
Thanks for changing. 😉 
LGTM

**Review by schnickalodeon — APPROVED** [review]
All fine - lg2m

**Pull request merged**